src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const checkIfIncludes = (string: string, query: string): boolean => {
+  return string.toLowerCase().includes(query.toLowerCase());
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(
```

Comment:
well done! you can put this function outside App
and don't forget to delete unnecessary comments)